### PR TITLE
Measure `getStaticProps`, `getServerSideProps`

### PR DESCRIPTION
**Pull request by Timer**
[NEXT-88]

[NEXT-88]: https://zeit.atlassian.net/browse/NEXT-88

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 7.8s | 8.2s | ⚠️ +335ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| index.html gzip | 936 B | 936 B | ✓ |
| link.html gzip | 946 B | 946 B | ✓ |
| withRouter.html gzip | 934 B | 934 B | ✓ |
| Overall change | 2.82 kB | 2.82 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 8s | 7.9s | -87ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _error.js gzip | 292 kB | 292 kB | ⚠️ +229 B |
| 404.html gzip | 1.34 kB | 1.34 kB | ✓ |
| hooks.html gzip | 975 B | 975 B | ✓ |
| index.js gzip | 292 kB | 292 kB | ⚠️ +313 B |
| link.js gzip | 300 kB | 299 kB | -534 B |
| routerDirect.js gzip | 298 kB | 299 kB | ⚠️ +692 B |
| withRouter.js gzip | 299 kB | 298 kB | -1.14 kB |
| Overall change | 1.48 MB | 1.48 MB | -437 B |

</details>

</details>


Commit: c437a463dc71dd6657afb78fd9c4feed40dbf4a0

File: packages/next/build/index.ts [hunk 1]
```diff
@@ -815,7 +815,11 @@ export default async function build(dir: string, conf = null): Promise<void> {
     eventBuildOptimize(pagePaths, {
       durationInSeconds: analysisEnd[0],
       staticPageCount: staticPages.size,
-      ssrPageCount: pagePaths.length - staticPages.size,
+      staticPropsPageCount: ssgPages.size,
+      serverPropsPageCount: serverPropsPages.size,
+      ssrPageCount:
```
Comment (Timer) [inline packages/next/build/index.ts]: This count has been wrong historically. Fixed now!

**Review by ijjk — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 8.7s | 8.9s | ⚠️ +259ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| index.html gzip | 936 B | 936 B | ✓ |
| link.html gzip | 946 B | 946 B | ✓ |
| withRouter.html gzip | 934 B | 934 B | ✓ |
| Overall change | 2.82 kB | 2.82 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 9.6s | 9.6s | -34ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _error.js gzip | 292 kB | 292 kB | -173 B |
| 404.html gzip | 1.34 kB | 1.34 kB | ✓ |
| hooks.html gzip | 975 B | 975 B | ✓ |
| index.js gzip | 292 kB | 293 kB | ⚠️ +525 B |
| link.js gzip | 300 kB | 299 kB | -849 B |
| routerDirect.js gzip | 299 kB | 298 kB | -662 B |
| withRouter.js gzip | 299 kB | 299 kB | -2 B |
| Overall change | 1.48 MB | 1.48 MB | -1.16 kB |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 8.7s | 8.8s | ⚠️ +172ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| index.html gzip | 936 B | 936 B | ✓ |
| link.html gzip | 946 B | 946 B | ✓ |
| withRouter.html gzip | 934 B | 934 B | ✓ |
| Overall change | 2.82 kB | 2.82 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| buildDuration | 9.5s | 9.3s | -137ms |
| nodeModulesSize | 56.3 MB | 56.3 MB | ⚠️ +398 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.76 kB | 5.76 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..67cb.js gzip | 9.69 kB | 9.69 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 55.3 kB | 55.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.64 kB | 6.64 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.3 kB | 51.3 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 18.9 kB | 18.9 kB | ✓ |
| Overall change | 18.9 kB | 18.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.js gzip | 1.07 kB | 1.07 kB | ✓ |
| _error.js gzip | 2.98 kB | 2.98 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 1.89 kB | 1.89 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.38 kB | 7.38 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _app.module.js gzip | 589 B | 589 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.27 kB | 5.27 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | Timer/next.js record-gsp-gssp | Change |
| - | - | - | - |
| _error.js gzip | 292 kB | 292 kB | -354 B |
| 404.html gzip | 1.34 kB | 1.34 kB | ✓ |
| hooks.html gzip | 975 B | 975 B | ✓ |
| index.js gzip | 292 kB | 292 kB | ⚠️ +639 B |
| link.js gzip | 300 kB | 299 kB | -976 B |
| routerDirect.js gzip | 299 kB | 298 kB | -1.1 kB |
| withRouter.js gzip | 298 kB | 299 kB | ⚠️ +1.29 kB |
| Overall change | 1.48 MB | 1.48 MB | -506 B |

</details>

</details>

**Pull request merged**